Use all four RX rings for the BE7000 source WLAN configuration.

--- a/internal/QCN9224_i.ini
+++ b/internal/QCN9224_i.ini
@@ -223,7 +223,7 @@
 dp_reo_rings_map=0x3
 
 [SINGLE_PHY_5G]
-dp_reo_rings_map=0x4
+dp_reo_rings_map=0xf
 
 [SINGLE_PHY_6G]
 dp_reo_rings_map=0x3
